spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -232,6 +232,8 @@ public MessageProperties toMessageProperties(AMQP.BasicProperties source, Envelo
 
 	private volatile int defaultTxSize = DEFAULT_TX_SIZE;
 
+	protected volatile boolean defaultDurableSubscription = true;
+
```

Comment:
>This affects only groups/taps.

I don't follow - it affects anyone who binds a consumer - we're switching the defaults on them again.

I am not convinced this should be pushed down to the `Binder` implementation(s) - some binders support it (RabbitMQ, JMS [future]) some don't.

It seems to me it's a common property that should be ignored by those who don't support it rather than making it a custom property of each binder that does support it.